### PR TITLE
MC-14148 Update API docs for stackpath serverless scripts

### PR DESCRIPTION
**Pull request by aaronalmeida**
#### Changes made

- Created the new section for Serverless Scripts
- Added the section for fetching a single and multiple entities

### CODE REVIEW

File: source/includes/stackpath/_scripts.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+## Scripts
+
+Deploy and manage Serverless Scripts used to interact with requests made to the site.
```
Comment (monil-panchal) [inline source/includes/stackpath/_scripts.md:3]: Not sure about the context of `request` to `site` here, but how about 
```
Deploy and manage Serverless Scripts for interacting with the sites
```
But if the original one fits well in the context, we are good to go 👍🏻

**Review by razine-bensari — APPROVED** [review]
looks good

File: source/includes/stackpath/_scripts.md (hunk 1 above)
Comment (aaronalmeida) [reply source/includes/stackpath/_scripts.md:3]: I got that from the Stackpath documentation https://support.stackpath.com/hc/en-us/articles/360001473706-Learn-About-Serverless-Scripting-Basics (second sentence) so it fits well within the context

**Review by monil-panchal — APPROVED** [review]
LGTM 🚀

**Pull request merged**